src/virtualenv/run/__init__.py

```diff
@@ -20,6 +20,12 @@ def run_via_cli(args):
     """
     session = session_via_cli(args)
     session.run()
+
+    logging.warn(
+        "Created virtual environment via %s, using %s",
+        six.ensure_text(str(session.creator)),
+        six.ensure_text(str(session.seeder)),
+    )
```

Comment:
Note that if #1568 is not accepted, the message here should also include the destination directory. (I don't really mind whether it does in the case where the user explicitly specifies the directory name, but it should if the default is used).

Reply:
the creator contains the destination directory 👍

Reply:
Ah, that's great then. Sorry, I didn't have enough time to check the details.

Reply:
Yea, I spent quite a few minutes figuring that out myself. :)

Reply:
Sad to hear that 😢